inlang/source-code/markdown/package.json

```diff
@@ -0,0 +1,49 @@
+{
+	"name": "@inlang/markdown",
+	"type": "module",
+	"version": "0.0.0",
+	"private": true,
+	"files": [
+		"./src",
+		"./dist"
+	],
+	"exports": {
+		".": "./dist/index.js",
+		"./custom-elements": "./dist/custom-elements/index.js",
+		"./css": "./dist/css/index.js"
+	},
+	"scripts": {
+		"build": "npx playwright install --with-deps chromium && npx tailwindcss -i ./src/css/style.css -o ./dist/css/style.css && npx prettier ./src --write && tsc --build",
```

Comment:
@floriandwt I am unsure if we should support markdown: 

- hundreds of dependencies
- requirement for a headless browser which slows down testing, ci/cd, install scripts, and more. 

Let's take out mermaid for now, and remove the unproportional amount of complexity entailed by mermaid support. 

I am looking at https://d2lang.com/ as a replacement because d2 can render without a headless browser, and has a nicer syntax. Unfortunately, no JS bindings exists yet.